metadata-io/src/main/java/com/linkedin/metadata/timeline/differ/DatasetPropertiesDiffer.java

```diff
@@ -19,39 +21,42 @@
 import static com.linkedin.metadata.timeline.differ.EditableDatasetPropertiesDiffer.*;
 
 
-public class DatasetPropertiesDiffer implements Differ {
+public class DatasetPropertiesDiffer implements AspectDiffer<DatasetProperties> {
   private static List<ChangeEvent> computeDiffs(DatasetProperties baseDatasetProperties,
-      @Nonnull DatasetProperties targetDatasetProperties, @Nonnull String entityUrn) {
+      @Nonnull DatasetProperties targetDatasetProperties, @Nonnull String entityUrn, AuditStamp auditStamp) {
     List<ChangeEvent> changeEvents = new ArrayList<>();
     String baseDescription = (baseDatasetProperties != null) ? baseDatasetProperties.getDescription() : null;
     String targetDescription = (targetDatasetProperties != null) ? targetDatasetProperties.getDescription() : null;
 
     if (baseDescription == null && targetDescription != null) {
       // Description added
       changeEvents.add(ChangeEvent.builder()
-          .target(entityUrn)
+          .entityUrn(entityUrn)
           .category(ChangeCategory.DOCUMENTATION)
-          .changeType(ChangeOperation.ADD)
+          .operation(ChangeOperation.ADD)
           .semVerChange(SemanticChangeType.MINOR)
           .description(String.format(DESCRIPTION_ADDED, targetDescription, entityUrn))
```

Comment:
Can you go ahead and reverse these while you're in here? Should be entityUrn, targetDescription

Reply:
yep!